example_configs/wildfly-10.yaml

```diff
@@ -0,0 +1,35 @@
+---
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames:
+  - "jboss.as:subsystem=datasources,data-source=*,statistics=*"
+  - "jboss.as:subsystem=datasources,xa-data-source=*,statistics=*"
+  - "jboss.as:subsystem=messaging-activemq,server=*,jms-queue=*"
+  - "jboss.as:subsystem=messaging-activemq,server=*,jms-topic=*"
+  - "jboss.as:subsystem=transactions"
+  - "jboss.as:subsystem=undertow,server=*,http-listener=*"
+  
+rules:
+  - pattern: "^jboss.as<subsystem=messaging-activemq, server=.+, jms-(?:queue|topic)=(.+)><>(.+):"
+    attrNameSnakeCase: true
+    name: wildfly_messaging_$2
+    labels:
+      name: $1
+
+  - pattern: "^jboss.as<subsystem=datasources, (?:xa-)*data-source=(.+), statistics=(.+)><>(.+):"
+    attrNameSnakeCase: true
+    name: wildfly_datasource_$2_$3
+    labels:
+      name: $1
+
+  - pattern: "^jboss.as<subsystem=transactions><>(number_of_.+):"
```

Comment:
I'd not include the number_of_ in the output name. Every metric is the number of something.